index.bs

```diff
@@ -5297,12 +5299,12 @@ created [=/request=] belongs to is [=transaction/aborted=] using the steps to
 
 
 <!-- ============================================================ -->
-## Running an upgrade transaction ## {#upgrade-transaction-steps}
+## Upgrading a database ## {#upgrade-transaction-steps}
 <!-- ============================================================ -->
 
 <div algorithm>
 
-To <dfn>run an upgrade transaction</dfn> with |connection| (a [=/connection=])
+To <dfn>upgrade a database</dfn> with |connection| (a [=/connection=])
 which is used to update the [=/database=], a new |version| to be set
```

Comment:
> which is used to update the database

This part seems a little superfluous maybe.

Reply:
Thanks. Simplified a bunch, now that "database" is clearly the object of the phrase.